packages/react/src/components/cookieConsent/consentsInModal/ConsentsInModal.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Modal } from '../modal/Modal';
+import { Content, Provider as CookieContextProvider } from '../CookieConsentContext';
+
+export function ConsentsInModal(props: { content: Content; cookieDomain?: string }): React.ReactElement | null {
```

Comment:
Since this is now the public api, it would be beneficial to type the props here, and give comments about them. Then they will show up in storybook controls tab description:

<img width="1397" alt="image" src="https://user-images.githubusercontent.com/2777633/165785967-05d338a5-70d2-4627-be38-faae27fcf190.png">

Reply:
List **all** object properties here? About 20 of them.